data/Arts2023Longitudinal.yaml

```diff
@@ -0,0 +1,3855 @@
+# yaml-language-server: $schema=.schema.yaml
+title: Longitudinal and quantitative fecal shedding dynamics of SARS-CoV-2, pepper
+  mild mottle virus, and crAssphage
+doi: 10.1128/msphere.00132-23
+description: >
+  The authors present longitudinal, quantitative fecal shedding data for SARS-CoV-2
+  RNA, pepper mild mottle virus (PMMoV) RNA, and crAss-like phage (crAssphage) DNA
+  from 48 COVID-19 patients. Abundances were quantified using (RT)-ddPCR assays targeting
+  the N and ORF1a genes. The data were contributed by the corresponding author.
+analytes:
+  stool_SARSCoV2_N:
+    description: >
+      Concentration of RNA of the N gene quantified using (RT)-ddPCR in stool samples.
+      The concentration were quantified in gene copies per dry weight of stool. The
+      limit of blank (LOB), determined as the upper 95% confidence limit of the negative
+      extraction control (30), ranged from 11.2 to 1,550 gc/mg-dry weight.
```

Comment:
What does the "(30)" mean here?

Reply:
The (30) is a reference from the original paper, which cites a study by Armbruster and Pry (2008) on the limit of blank, limit of detection, and limit of quantitation. I included it to maintain consistency with the original source but forgot to remove it for clarity. I'll go ahead and delete it to avoid confusion. Thanks for catching that!